dali/base/dadfs.cpp

```diff
@@ -8239,20 +8239,22 @@ class CNamedGroupStore: implements INamedGroupStore, public CInterface
 
 };
 
-static CNamedGroupStore *groupStore = NULL;
+static std::atomic<CNamedGroupStore *> groupStore{nullptr};
```

Comment:
There seems to be a mixture of fixes here.  Either the variable should be atomic, or it should be accessed within a critical section.  No need to do both.